ansible_base/lib/dynamic_config/dynamic_settings.py

```diff
@@ -139,6 +139,8 @@
     SOCIAL_AUTH_STRATEGY = "ansible_base.authentication.social_auth.AuthenticatorStrategy"
     SOCIAL_AUTH_LOGIN_REDIRECT_URL = "/"
 
+    ANSIBLE_BASE_SOCIAL_AUDITOR_FLAG = "is_system_auditor"
```

Comment:
Should we change this to default to `is_platform_auditor` as a "going forward" approach?
Also, we should add this to the docs for authentication to let people know they need to set it if required?

Reply:
My logic here is that "is_system_auditor" isn't going away, it just gives an indicator of "local" permissions, not permissions that are shared with some other system. And maybe we don't migrate it at all, and move towards role mappings instead. So I still like this as the default. But maybe it should be added in docs.